spec/features/checking_in_spec.rb

```diff
@@ -14,24 +11,20 @@
     fill_in 'Day', with: '1 January, 2016'
     fill_in 'pick-a-time', with: '4:30 PM'
     select 'Start Shift', from: 'Action'
-    find(:css, '#check_in_form_signature', visible: false).set 'my signature'
+    find('#check_in_form_signature', visible: false).set 'my signature'
     click_button 'Save'
 
-    expect(Shift.count).to eq(shift_count + 1)
-    expect(ShiftEvent.count).to eq(shift_event_count + 1)
-    expect(Volunteer.count).to eq(volunteer_count + 1)
-
-    shift_event = ShiftEvent.first
-    expect(shift_event.action).to eq('start_shift')
-    expect(shift_event.occurred_at).to eq(Time.zone.local(2016, 1, 1, 16, 30))
+    expect(page).to have_content 'Check in has been saved.'
```

Comment:
So among the the following options

1) keep this part as it was
2) keep the changes in the PR
3) log in through the admin dashboard & verify

I feel like 3 is a little overkill (or at least too much to test in one spec), and I'm guessing you agree, but I can't quite tell based on how you wrote it. I still feel like 2 is the better option, and after having reviewed the changes to the form spec again, I think the only change in coverage is that we aren't checking the shift event action (because we'd essentially just be checking delegation and the assignment operation).

That said, I know it can't be kinda risky to refactor specs, so I understand if you'd prefer option 1. 

Just let me know what you think :)